src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -2029,5 +2029,42 @@ def test_network_watcher(self, resource_group, storage_account):
         self._network_watcher_troubleshooting()
 
 
+class ServiceEndpointScenarioTest(ScenarioTest):
+
+    #@ResourceGroupPreparer(name_prefix='network_service_endpoint_scenario_test')
+    def test_network_service_endpoints(self):#, resource_group):
+
+        self.kwargs.update({
+            'policy': 'policy1',
+            'pd_name': 'storage-def',
+            'sub': self.get_subscription_id(),
+            'vnet': 'vnet1',
+            'subnet': 'subnet1',
+            'rg': 'tjp-vm'
+        })
+
+        self.cmd('network service-endpoint list -l westus')
+
+        # test policy CRUD
+        self.cmd('network service-endpoint policy create -g {rg} -n {policy} --tags test=best')
+        self.cmd('network service-endpoint policy update -g {rg} -n {policy} --tags test=nest')
```

Comment:
Please add some validations here for this test case.
Also, are you able to get the wheel file from associated Python SDK PR?  If yes, you can put under the `privates` folder, then CI can run normally.

Reply:
Yes, I will add validations. I'm not concerned right now about the CI.